src/containers/course-sections-list/CourseSectionsList.styles.ts

```diff
@@ -26,7 +26,8 @@ export const styles = {
     fontSize: '30px',
     transform: 'rotate(90deg)',
     color: 'primary.400',
-    cursor: 'pointer'
+    cursor: 'grab',
+    touchAction: 'none'
```

Comment:
I have noticed that these styles are quite common inside drag icons. Maybe we can adjust `DragHandle.tsx` and add these styles only in one place to all handlers or just create some common style to reuse it? What do you think about it?

Reply:
i created a separate file for dragHandle style. Fixed